### PR TITLE
numpy 1.19.1

### PR DESCRIPTION
**Pull request by dtrodrigues**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dtrodrigues** [comment]
```
Error: 1 failed steps!
brew test --verbose adios2

 ==> /usr/local/opt/python@3.8/bin/python3 -c import adios2
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/adios2/__init__.py", line 1, in <module>
    from .adios2 import *
ImportError: dlopen(/usr/local/lib/python3.8/site-packages/adios2/adios2.cpython-38-darwin.so, 2): Symbol not found: _ofi_process_vm_writev
  Referenced from: /usr/local/opt/libfabric/lib/libfabric.1.dylib
  Expected in: flat namespace
 in /usr/local/opt/libfabric/lib/libfabric.1.dylib
Error: adios2: failed
```

File: Formula/libfabric.rb [hunk 1]
```diff
@@ -17,6 +17,12 @@ class Libfabric < Formula
   depends_on "automake" => :build
   depends_on "libtool"  => :build
 
+  # https://github.com/ofiwg/libfabric/pull/6109 is merged upstream, remove this with next release
+  patch do
+    url "https://github.com/ofiwg/libfabric/pull/6109.diff?full_index=1"
```
Comment (SeekingMeaning) [inline Formula/libfabric.rb:22]:
```suggestion
    url "https://github.com/ofiwg/libfabric/commit/85c9732fd95f9970f5bcf793ca580d45ed7418f2.diff?full_index=1"
```

Also:

- Should this be in a separate PR?
- Does this need a revision bump?

Comment (Bo98) [reply Formula/libfabric.rb:22]: > Does this need a revision bump?

Yes, this patch is one that would warrant a revision bump.

**Comment by dtrodrigues** [comment]
Depends on https://github.com/Homebrew/homebrew-core/pull/58437, will rebase and force-push once that's merged.

**Review by SeekingMeaning — APPROVED** [review]
